todo/src/Reducers/toggleReducer.js

```diff
@@ -0,0 +1,21 @@
+import { TOGGLE_TODO } from '../Actions/actionTypes'
+
+const _defaultState = {};
+
+const toggleReducer = (state = _defaultState, action ) => {
```

Comment:
Move all this logic to the todoReducer, and all of this would be a case in the reduce, like `TOGGLE_TODO`